spec/lib/psulib_traject/workers/hourly_indexer_spec.rb

```diff
@@ -1,14 +1,42 @@
 # frozen_string_literal: true
 
 require 'spec_helper'
+require 'sidekiq/testing'
 
-RSpec.describe PsulibTraject::Workers::HourlyIndexer, skip: ci_build? do
-  let(:indexer) { described_class.new }
+Sidekiq::Testing.fake!
```

Comment:
I'm wondering if this shouldn't live in `spec/support/sidekiq.rb` too. Does it need to be "unfaked" at some point?

Reply:
I'd think if it needs to be unfaked we can re-arrange at that point.